.github/workflows/update_repos.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       - name: Find repos
         id: find_repos
         run: |
-          if [ "${{ inputs.repos }}" = "__all_cookiecuttered_repos__" ]
+          if [ "${{ inputs.repos || '__all_cookiecuttered_repos__' }}" = "__all_cookiecuttered_repos__" ]
```

Comment:
This is the actual bug fix. Just turn an empty string into `'__all_cookiecuttered_repos__'`